File: Nível 1/2388 - Tacógrafo/main.lua
valor = tonumber(io.read())
kms = 0

for i = 1, valor do 
    intervaloTempo = io.read()
    x1, x2 = intervaloTempo:match("(%S+)%s+(%S+)")
    x1, x2 = tonumber(x1), tonumber(x2)
    formula = x2 * x1
    kms = kms + formula
end

print(kms)

File: Nível 1/2388 - Tacógrafo/main.py
valor = int(input("A: "))
kms = 0

for i in range(1, valor + 1):
    intervaloTempo = input("B: ")
    x1,x2 = intervaloTempo.split(" ")

    x1 = int(x1)
    x2 = int(x2)

    formula = x2 * x1
    kms += formula

print(kms)